### PR TITLE
improve design of research plan attachments

### PR DESCRIPTION
**Pull request by JoachimTillessen**
- [ ] rather 1-story 1-commit than sub-atomic commits

- [ ] commit title is meaningful =>  git history search

- [ ] commit description is helpful => helps the reviewer to understand the changes

- [ ] code is up-to-date with the latest developments of the target branch (rebased to it or whatever) => :fast_forward:-merge for linear history is favoured

- [ ] added code is linted

- [ ] tests are passing (at least locally): we still have some random test failure on CI. thinking of asking spec/examples.txt to be commited

- [ ] in case the changes are visible to the end-user,  video or screenshots should be added to the PR => helps  with user testing

- [ ] testing coverage improvement is improved.

- [ ] CHANGELOG :  add a bullet point on top (optional: reference to github issue/PR )

- [ ] parallele PR for documentation  on docusaurus  if the feature/fix is tagged for a release

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -17,12 +17,12 @@
 @import "global-styles/utilities";
 
 // file css rules that are targeted at specific components under components
+@import "components/AttachmentList";
 @import "components/ButtonGroupToggleButton";
 @import "components/search-modal";
 @import "components/search-results";
```
Comment (beque) [inline app/assets/stylesheets/application.scss:24]: General question: The files should be written in camelcase?

**Review by beque — APPROVED** [review]
Looks good, just one general questions
But I can't see the overlay (when you are over an attachment) anymore!

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (JoachimTillessen) [reply app/assets/stylesheets/application.scss:24]: I'm accustomed to give the scss files the exact same name as the corresponding component. But I'm not sure weather that's an obvious pattern to others. Especially in this project where naming has plenty of inconsistencies. What do you think?

**Comment by JoachimTillessen** [comment]
> Looks good, just one general questions But I can't see the overlay (when you are over an attachment) anymore!

You're right! Looks like I broke it. I gotta fix this.

**Pull request merged**